server/bin/pbench-cull-unpacked-tarballs.py

```diff
@@ -332,58 +329,23 @@ def main(options):
 
     logger = get_pbench_logger(_NAME_, config)
 
-    archive = config.ARCHIVE
-    archive_p = os.path.realpath(archive)
-
-    if not archive_p:
-        logger.error("The configured ARCHIVE directory, {}, does not exist", archive)
-        return 3
-
-    if not os.path.isdir(archive_p):
-        logger.error(
-            "The configured ARCHIVE directory, {}, is not a valid directory", archive
-        )
-        return 4
+    archivepath = config.ARCHIVE
 
-    incoming = config.INCOMING
-    incoming_p = os.path.realpath(incoming)
-
-    if not incoming_p:
-        logger.error("The configured INCOMING directory, {}, does not exist", incoming)
+    incoming = Path(config.INCOMING)
+    incomingpath = config.get_valid_dir_option("INCOMING", incoming, logger)
+    if incomingpath == 1:
```

Comment:
Return value -> None allows this to be simplified as in backuppath in pbench-backup-tarballs.py